### DISCUSSION TITLE
Icons missing despite installing nerd fonts mentioned in the readme

### DISCUSSION THREAD

**Discussion by indeedwatson** [alexanderjeurissen/ranger_devicons#85] [Personal Support]
Previously I had the complete nerd fonts package installed, but that took about 9GB of cache, so I stopped using it.

Following the readme, I installed Source Code Pro patched NERDfont (nerd-fonts-source-code-pro from AUR), and I also have ttf-nerd-fonts-symbols. However, some icons are still missing:

![1616086621](https://user-images.githubusercontent.com/10749504/111666488-85b50300-87f2-11eb-8c2e-80220f96126d.png)

**Comment by alexanderjeurissen** [comment]
hi can you provide some more info about your environment ?

e.g.

OS + version: [e.g. Mac OS Catalina 10.15.6]
shell [e.g. fish, zsh]
shell version [e.g. 3.1.2]

and about your Ranger config ?:

version / commit [e.g. HEAD]
output of tree ~/.config/ranger [can be installed on most OS using default package manager e.g. apt-get install tree]

**Comment by vitorfrois** [comment]
I think I have the same problem. Still looking for solutions
Manjaro i3wm, URxvt shell with fish
This is how ranger looks like:
![2022-01-18_15-42](https://user-images.githubusercontent.com/46361092/149999109-0b963d42-df33-4930-8d9f-2dc485099d1b.png)
